src/test/runTest.ts

```diff
@@ -17,7 +17,11 @@ async function main() {
 			version: "insiders",
 			extensionDevelopmentPath,
 			extensionTestsPath,
-			launchArgs: ["--disable-gpu", "--disable-keytar"]
+			launchArgs: [
+				"--disable-gpu",
+				"--disable-keytar",
+				'--password-store=basic',
```

Comment:
Trying this as suggested on the other thread

```suggestion
				'--password-store=basic_text',
```